enigma/src/main/java/cuchaz/enigma/EnigmaProject.java

```diff
@@ -1,346 +1,348 @@
-package cuchaz.enigma;
```

Comment:
What's going on with the diffs spanning entire files with most of the lines not actually changing? Please fix this somehow.

Reply:
They're whitespace changes. (I'd assume LF<=>CRLF, but I haven't actually checked.)

Reply:
I think it's line ending. I have `git config core.autocrlf` turned off for jdk

Reply:
Yup, it actually is. liach said he turned off core.autocrlf for another repo. (fyi that can be changed per repo with git config --local)

Still, we don't want crlf line endings in the repo so this should still be fixed.